packages/dataviews/package.json

```diff
@@ -24,6 +24,20 @@
 	},
 	"main": "build/index.js",
 	"module": "build-module/index.js",
+	"exports": {
+		".": {
+			"types": "./build-types/index.d.ts",
+			"import": "./build-module/index.js"
+		},
+		"./wp": {
+			"types": "./build-types/index.d.ts",
+			"default": "./wp/index.js"
```

Comment:
Could we store the build results in a `build-wp` subfolder instead? It would be nice and more intuitive if all transient and git-ignored build directories had the `build-*` prefix.

Reply:
Good call.